nsxt/resource_nsxt_policy_project.go

```diff
@@ -112,16 +112,24 @@ func resourceNsxtPolicyProject() *schema.Resource {
 			"vpc_deployment_scope": {
 				Type:        schema.TypeList,
 				Optional:    true,
-				Computed:    true,
+				Computed:    false,
 				Description: "Project Vpc network Deployment Scope",
 				MaxItems:    1,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
+						"default_span_path": {
+							Type:         schema.TypeString,
+							Description:  "Policy path of the Cluster based default Span object of type NetworkSpan",
+							ValidateFunc: validatePolicyPath(),
+							Default:      "/infra/network-spans/default",
+							Optional:     true,
+							Computed:     false,
```

Comment:
Since NSX API may populate the default span, should this be computed?

Reply:
Also, `Computed` is false by default, you could instead drop the `Computed` attribute from these schema attrs.